opengrok-indexer/src/test/java/org/opengrok/indexer/search/SearchEngineTest.java

```diff
@@ -55,7 +61,9 @@ class SearchEngineTest {
     @BeforeAll
     static void setUpClass() throws Exception {
         repository = new TestRepository();
-        repository.create(HistoryGuru.class.getResource("/repositories"));
+        URL url = HistoryGuru.class.getResource("/repositories");
+        repository.createEmpty();
+        repository.copyDirectoryWithUniqueModifiedTime(Path.of(url.toURI()), Path.of(repository.getSourceRoot()));
```

Comment:
Please add `assertNotNull` for the `url` before the `toURI()` conversion like it is done in some other test classes to avoid code smells reported e.g. by SonarQube.